pkg/exporter/sumologicexporter/sender_test.go

```diff
@@ -1032,82 +1026,6 @@ func TestInvalidPipeline(t *testing.T) {
 	assert.EqualError(t, err, `unknown pipeline type: invalidPipeline`)
 }
 
-func TestSendCompressGzip(t *testing.T) {
```

Comment:
The sender module no longer has any compression logic, instead the compression encoding is now set in the `HttpClientSettings`